### PR TITLE
Added info about creating a empty section

### PR DESCRIPTION
**Pull request by enkelmedia**
Added a code sample based on this thread: https://github.com/umbraco/Umbraco.CMS.Backoffice/issues/841 I guess that it might need to be improved but figured it was better than nothing =D

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md [hunk 1]
```diff
@@ -25,6 +25,22 @@ This page is a work in progress. It will be updated as the software evolves.
...
+const section : ManifestSection = {
+    "type": "section",
+    "alias": "Empty.Section",
+    "name" : 'Empty Section',
+    element : () => import('./empty-section.element.js'),
+    meta : {
+        label : 'Foo',
+        pathname : 'empty-section'
+    }
+}
```
Comment (alina-tincas) [inline 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md:41]: I keep trying to add this example in umbraco-package.json however I cannot find a way to make this code work 🤔 

Can you please provide a full example from your package manifest to see how one can reproduce this on a new installation?

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @enkelmedia thank you for the PR 🙌 

In order to reproduce this code, can you please share a little bit more information? 🙏

File: 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md (hunk 1 above)
Comment (enkelmedia) [reply 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md:41]: Ohh, sorry, this was added using a ”entity point” so it’s actually typescript code. Sorry for the lack of context.

I’ll see if I can update it to make more sense.

**Comment by enkelmedia** [comment]
Hi again @alina-tincas 

I just pushed an update and while writing this i felt that it might not be the "first thing" that some one to talk about. I was playing with this and when I realized that configuring a custom element via the `element` property will disable dashboards and sectionViews I feel that it might not be that useful?

In any case, I do still believe that it's a good idea to recommend people using a entryPoint extension and configure additional extensions using TypeScript when creating more complex extensions like a section.

What do you think? Should we remove the part about element and just keep the entryPoint stuff?

File: 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md [hunk 2]
```diff
@@ -11,20 +11,41 @@ This page is a work in progress. It will be updated as the software evolves.
 <figure><img src="../../../.gitbook/assets/section.svg" alt=""><figcaption><p>Section</p></figcaption></figure>
 
 **Manifest**
+When creating a new section it's a good idea to use a [Entry Point](../extension-types/entry-point)-extension in your [Umbraco Package Manifest](../../package-manifest) to get better control over all the additional extensions required for the new section.
```
Comment (alina-tincas) [inline 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md:14]:
```suggestion
When creating a new section it's recommended to use a [Entry Point](../extension-types/entry-point)-extension in your [Umbraco Package Manifest](../../package-manifest). This is to get better control over all the additional extensions required for the new section.
```

File: 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md [hunk 3]
```diff
@@ -11,20 +11,41 @@ This page is a work in progress. It will be updated as the software evolves.
...
+    name : 'Empty Section',
+    element : () => import('./empty-section.element.js'),
+    meta : {
+        label : 'Empty Section',
+        pathname : 'empty-section'
+    }
+}
+```
+
+Notice that if you pass an element like this, the default behavior of loading dashboards and sectionViews will be disabled.
```
Comment (alina-tincas) [inline 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md:47]:
```suggestion
{% hint style="warning" %}
If you pass an element like this, the default behavior of loading dashboards and sectionViews will be disabled.{% endhint %}
```

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Hi @enkelmedia thank you for elaborating 🙌 

Now I understand the use case and agree that for more complex examples entry points can be used 👍 

I will leave those examples as they are now and later on when we look into this page again to add more content we will see if it makes sense to have both examples and remove the one with the element 💪 

Thank you again for contributing!

**Pull request merged**